### ISSUE TITLE
Copyright violation

### ISSUE THREAD

**Issue by Poussinou** [moezbhatti/qksms#786]
Hi @moezbhatti 

I found [this app](https://play.google.com/store/apps/details?id=com.msun.cleansms) and also [this one](https://play.google.com/store/apps/details?id=com.smstext.messaging) on the Google Play Store. This guys just took your source code, added some ads in it (or sell it for 20 euros... !!) and put it on the Store, without warning this is free software... They does'nt respect the GPL v3 at all...

As you are the owner of the source code, please feel free to open a request to google [here](https://support.google.com/legal/contact/lr_dmca?product=googleplay) to remove the fake apps from the Store ;)

Don't forget to tell Google that:

* The app is licensed under GNU GPL v3 license and this guys are violating it;
* They are falsely claiming they made the app;
* Provide a link to this repo and tell them this app is open-source;
* Provide screenshots of your own app.

**Comment by moezbhatti** [comment]
Thanks for the heads up! I'll make sure to follow up about this

**Comment by frogoscar** [comment]
This proves Google Play's check system is just useless.

**Comment by moezbhatti** [comment]
[One more](https://play.google.com/store/apps/details?id=com.msun.cleanmessaging&hl=en)
Thanks to @m2ck for finding it

**Comment by Poussinou** [comment]
@moezbhatti I found another one [here](https://play.google.com/store/apps/details?id=com.sms.mms.pro) !
Can you report it?

**Comment by Poussinou** [comment]
The "developer" of [this copy](https://play.google.com/store/apps/details?id=com.msun.cleanmessaging) tells me that he has an agreement with you to copy your app, is that right?

![image](https://user-images.githubusercontent.com/19646549/32742008-6cfd9c26-c8a8-11e7-8840-e3918ef58c96.png)

**Comment by moezbhatti** [comment]
@Poussinou thanks for following up on that! This is something that I should have brought up as soon as this issue was created - I apologize for the lack of communication on my part

The developer does in fact have an agreement with me. He reached out shortly after QKSMS was open sourced in 2015, and asked for permission to use the code in his closed-source application. I negotiated a deal with him to allow him to use the code as-is in his app.

Aside from this, there was one other developer that I granted this permission for. However this was for an enterprise application and isn’t publicly available on the Play Store.

Again, sorry for not mentioning this earlier. If you have any other questions please let know!

**Comment by Poussinou** [comment]
Hum then if you have a deal with him, I'll remove his application from my list of copies of QKSMS, and also the screenshot above.

Do not apologize, it's your app and you communicate what you want when you want ;)

But please report the others copies, it will help me in my hunt for fake developers <3

**Comment by Poussinou** [comment]
@moezbhatti Two more here:
* https://play.google.com/store/apps/details?id=com.dinesh.DineshSMS
* https://play.google.com/store/apps/details?id=com.sharetheworld.quicksms

**Comment by moezbhatti** [comment]
Another one: https://play.google.com/store/apps/details?id=com.extecitek.titrasms

**Comment by Poussinou** [comment]
Thanks for reporting 👍

**Comment by Poussinou** [comment]
We have another one here: https://play.google.com/store/apps/details?id=com.quick.sms.mms

**Comment by moezbhatti** [comment]
Another one - https://play.google.com/store/apps/details?id=com.trentpierce.aurasms&hl=en_US

This one at least credits QKSMS and links to the original source, but it doesn't provide its own source, so it's still in violation of GPL

**Comment by Poussinou** [comment]
You found this app by chance?

**Comment by moezbhatti** [comment]
I have a daily email from mention.net which lets me know where people are mentioning QKSMS online - the app was part of that

**Comment by sm4rk0** [comment]
5,000,000+ installs for com.promessage.message !?

**Comment by moezbhatti** [comment]
@sm4rk0 yep. I checked out the app and it looks like they never even changed the email address for the MMS support.

Now I know where those 49,328 support emails came from...

<img width="200" alt="screen shot 2018-08-16 at 10 42 01 am" src="https://user-images.githubusercontent.com/4358785/44198191-072aed00-a141-11e8-86ee-6c82bee329d2.png">

**Comment by Poussinou** [comment]
Another one here ? https://play.google.com/store/apps/details?id=com.messaging.quicksms

**Comment by moezbhatti** [comment]
Another one https://play.google.com/store/apps/details?id=com.messenger.vpsms.multi.theme&hl=en

**Comment by Poussinou** [comment]
https://play.google.com/store/apps/details?id=com.message.smsmms

**Comment by moezbhatti** [comment]
This one seems to have customized their UI, but I'm receiving their support emails, so I think they may be using the QKSMS under the hood

https://play.google.com/store/apps/details?id=messages.textnow.textting.free

**Comment by Poussinou** [comment]
https://play.google.com/store/apps/details?id=org.mindapps.smsgo

**Comment by moezbhatti** [comment]
https://play.google.com/store/apps/details?id=free.textting.messages.sms.mms.free

**Comment by moezbhatti** [comment]
https://play.google.com/store/apps/details?id=truecaller.caller.callerid.name.phone.dialer

**Comment by moezbhatti** [comment]
https://play.google.com/store/apps/details?id=messenger.text.now

**Comment by moezbhatti** [comment]
https://play.google.com/store/apps/details?id=com.sunny.ger

**Comment by moezbhatti** [comment]
https://play.google.com/store/apps/details?id=smart.messages.smsmanager.textra.smsmessenger

**Comment by hbhatti** [comment]
I can't believe how blatant that is.

What recourse do you have?
